README.md

```diff
@@ -44,3 +44,5 @@ After that, you’ll draft emails and send them to your coach for review. Since
 This might sound like a lot of work, and in fact, it is. But when you’re prepared, focused, and have a plan of action, you’ll stop thinking of it as a chore and instead just one big exciting step to get you closer to where you want to be!
 
 <p class='util--hide'>View <a href='https://learn.co/lessons/careers-job-search-tracker'>Careers Job Search Tracker</a> on Learn.co and start learning to code for free.</p>
+
+<p class='util--hide'>View <a href='https://learn.co/lessons/careers-job-search-tracker'>Job Search Action Plan </a> on Learn.co and start learning to code for free.</p>
```

Comment:
@henrymly when you're resolving conflicts, keep line 48 and remove line 46, since it looks like we've updated the name of this lesson 😄